eth/backend.go

```diff
@@ -330,6 +330,50 @@ func (s *Ethereum) Etherbase() (eb common.Address, err error) {
 	return common.Address{}, fmt.Errorf("etherbase must be explicitly specified")
 }
 
+// isLocalAccount checks whether the specified address is a local account.
+//
+// This function is used during block chain reorg checking to determine
+// whether a block is mined by local accounts. We regard two types of
+// accounts as local account: etherbase and accounts specified via
+// `txpool.locals` flag.
+func (s *Ethereum) isLocalAccount(addr common.Address) bool {
```

Comment:
Perhaps lets call this `isMinerAccount`. It might be clearer that not all accounts are taken into consideration, rather only those that are considered miners.